<launch>

  <node pkg="tf" type="static_transform_publisher" name="test_new_tf" args="0 0 0 0 -1.5708 1.5708 camera_color_optical_frame child_camera_color_optical_frame 100" />
  <node pkg="cylinder_detection" type="cylinder_detection" name="cylinder_detection" output="screen">
      launch-prefix="xterm -e gdb --args"
      <param name="base_frame" value="/bubbleRob" />
      <param name="max_range" value="2.0" />
      <param name="n_samples" value="10000" />

      <param name="tolerance" value="0.005" />
      <remap from="/cylinder_detection/scans" to="/depth_registered/points"/>
  </node>
</launch>
